react_client/src/api/expressQueryAPI.jsx

```diff
@@ -22,7 +22,7 @@ import axios from "axios";
 export default async function expressQueryAPI(query) {
   try {
     return await axios
-      .post("http://localhost:5523/queries", {
+      .post("/api/queries", {
```

Comment:
now we do a proper rewrite of the api to the running localhost server. this may seem minor but it's preventing a whole slew of CORS and cookie issues but serving the app and the api from the same exact url